### PR TITLE
CI: Upload video artifacts from e2e tests

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Uploads video artifacts produced by e2e tests:

image

refs
- https://learn.microsoft.com/en-us/azure/devops/pipelines/artifacts/pipeline-artifacts?view=azure-devops&tabs=yaml#publish-artifacts
- https://learn.microsoft.com/en-us/azure/devops/pipelines/artifacts/pipeline-artifacts?view=azure-devops&tabs=yaml-task#publish-artifacts

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [n/a] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @carmocca @akihironitta @borda

### CODE REVIEW

**Comment by Borda** [comment]
@akihironitta, do you know if we can somehow upload artifacts?
I think it worked a while back, but not sure why it is not working anymore... :(

**Review by carmocca — APPROVED** [review]
Low confidence approval

File: .github/workflows/ci-schema.yml [hunk 1]
```diff
@@ -8,4 +8,8 @@ on:
 
 jobs:
   check:
-    uses: Lightning-AI/devtools/.github/workflows/check-schema.yml@v0.1.0
+    uses: Lightning-AI/utilities/.github/workflows/check-schema.yml@v0.4.0
```
Comment (Borda) [inline .github/workflows/ci-schema.yml:11]:
```suggestion
    uses: Lightning-AI/utilities/.github/workflows/check-schema.yml@v0.4.1
```

**Comment by Borda** [comment]
awesome, thank you @akihironitta!

**Review by otaj — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]

**Pull request merged**